yarn.lock

```diff
@@ -1202,6 +1202,13 @@
   dependencies:
```

Comment:
you will have to regenerate this

- undo all the changes here
- pull the latest main
- run `yarn install`